lib/draw.c

```diff
@@ -276,6 +279,20 @@ mesh rotateMeshAroundZ(mesh inputMesh, const double angle)
     return inputMesh;
 }
 
+vector calculateTriangleNormal(triangle inputTri)
+{   
+    vector U,V,normal;
+    
+    U = (subVec(inputTri.p[1],inputTri.p[0]));
+    V = (subVec(inputTri.p[2],inputTri.p[1]));
+
+    normal.x = (U.y * V.z) - (U.z * V.y);
+    normal.y = (U.z * V.x) - (U.x * V.z);
+    normal.z = (U.x * V.y) - (U.y * V.x);
```

Comment:
pull this out to be a crosss product function

Reply:
done :)